app/src/main/java/com/github/se/wanderpals/MainActivity.kt

```diff
@@ -78,7 +78,8 @@ class MainActivity : ComponentActivity() {
                       SessionManager.setUserSession(
                           userId = uid,
                           name = account.displayName ?: "",
-                          email = account.email ?: "")
+                          email = account.email ?: "",
+                          profilePhoto = it.result.user?.photoUrl.toString())
```

Comment:
Great initialisation of profilePhoto when you can retrieve something from the login